core/src/main/res/values/strings.xml

```diff
@@ -187,6 +187,7 @@
         <item quantity="other">%d subscriptions updated.</item>
     </plurals>
     <string name="add_to_folder">Add to folder</string>
+    <string name="rename_folder_label">Rename folder</string>
```

Comment:
Could you please rename this to "tag"? The feature was recently renamed to "tags" instead of "folders" (see #5381)

While you are at it, could you please also update the string+name above it? Please only change the English strings file, not the translations.